x/auth/keeper/keeper.go

```diff
@@ -274,3 +283,17 @@ func (ak AccountKeeper) getBech32Prefix() (string, error) {
 
 	return bech32Codec.bech32Prefix, nil
 }
+
+// SetParams sets the auth module's parameters.
+func (ak AccountKeeper) SetParams(ctx context.Context, params types.Params) error {
```

Comment:
is this for backwards compatibility?

Reply:
Reasons for keeping this method were two:
- The Set also does a param validation check.
- Not breaking Keeper API.